src/stratigraphy/metadata/elevation_extraction.py

```diff
@@ -197,4 +193,8 @@ def extract_elevation(self) -> ElevationInformation | None:
         for page in self.doc:
             page_number = page.number + 1  # page.number is 0-based
 
-            return self.extract_elevation_from_bbox(page, page_number)
+            # Extract the elevation information from the page
+            # Make the search area a bit larger than the page itself - This is needed for "267123120-bp.pdf" where the
+            # elevation is located outside the page boundaries apparently.
+            search_area = page.rect * 1.25
+            return self.extract_elevation_from_bbox(page, page_number, search_area)
```

Comment:
This should not be necessary. The bbox is an optional param in this method, it should correctly take the whole page when you omit this param (None value).